Makefile

```diff
@@ -52,7 +52,7 @@ publish-controller-image:  ## docker push a container image for SERVICE
 	@echo $(DOCKER_PASSWORD) | docker login -u $(DOCKER_USERNAME) --password-stdin
 	./scripts/publish-controller-image.sh $(AWS_SERVICE)
 
-build-controller: build-ack-generate ## Generate controller code for SERVICE
```

Comment:
any reason to remove this prerequisite?

Reply:
I thought that it will not work for contributors that installed `ack-generate` using `go get` but apparently it will ... https://github.com/aws-controllers-k8s/community/blob/main/Makefile#L33

Reply:
@A-Hilaly you're up way too late :)

If you can add the build-controller target back in, that would be great. I'll approve this after.